src/lib/config/default/config-schema.json

```diff
@@ -106,6 +105,19 @@
             ],
             "type": "string"
         },
+        "DetailName": {
+            "enum": [
+                "level",
+                "message",
+                "requestReceivedAt",
+                "responseSentAt",
+                "responseTimeMs",
+                "stacktrace",
+                "writtenAt",
+                "writtenTs"
+            ],
+            "type": "string"
+        },
```

Comment:
We can extend this enum, if required. However, if we want to support custom _DetailFunctions_, which could be configured by users to add additional functionality, we might need to change this approach and allow any string values here. However, this would come at the cost of config validation for this source type.